### DISCUSSION TITLE
Question about `anyOf` for nullable field

### DISCUSSION THREAD

**Discussion by dephiros** [json-schema-faker/json-schema-faker#766] [Q&A] [answered]
I start to adopt this library and Typebox inside our company and one of the usage is to generate fixture for testing from JSON Schema
One of the field that I am struggling with is nullable field. An example of how we would represent that with Typebox:
```
const ObjSchema = Type.Object({
  firstName: Type.String({
    faker: 'name.firstName',
  }),
  lastName: Type.Union([
    Type.String({
      faker: 'name.lastName'
    }),
    Type.Null()
  ])
});
```
which would generate this schema:
```
{
  "type": "object",
  "properties": {
    "firstName": {
      "type": "string",
      "faker": "name.firstName"
    },
    "lastName": {
      "anyOf": [
        {
          "type": "string",
          "faker": "name.lastName"
        },
        {
          "type": "null"
        }
      ]
    }
  }
}
```
which makes sense to me. The problem is that `json-schema-faker` does not seem to understand `anyOf` but understand `type: ["string", "null"]` which I also see as a valid way to represent nullable string according to JSON Schema.

Any suggestion to handle this case nicely from json-schema-faker end?

**Comment by pateketrueke** [comment]
I found the resulting schema working on my side, which version/options are you using?

**Reply by dephiros** [reply, marked as answer]
Thanks for checking 🙏. It actually work as expected(randomly select one of the anyOf). I think I might have messed something up previously
